write-config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# expects these environment variables:
+#
+# - `ELASTICSEARCH_ENDPOINT`: URL to elasticsearch
+
+# Our index name
+INDEX=.kibana
+
+# Check env variable
+if [ "$ELASTICSEARCH_ENDPOINT" == "" ]; then
```

Comment:
Use `${VARIABLE}` notation as a safer Variable expansion in bash

https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables